doc/manual/judging.rst

```diff
@@ -9,10 +9,13 @@ The flow of an incoming submission is as follows.
    checks, or accepted and stored as a *submission*.
 #. The first available *judgehost* compiles, runs and checks
    the submission. The outcome and outputs are stored as a
-   *judging* of this submission. Note that judgehosts may be
-   restricted to certain contests, languages and problems, so that it can be
-   the case that a judgehost is available, but not judging an available
-   submission.
+   *judging* of this submission. Note that it is possible to
+   parallelize this when only one submission is waiting.
+#. In case that multiple submissions are not judged yet the
+   load will be spread over all judgehosts. There is a slight delay
```

Comment:
Suggested phrasing (how do I do this in this github UI directly?):
To avoid starving other teams from judgehost resources, submissions from teams that submit while they have other submissions in the judge queue will be penalized by downgrading their judging priority.

Reply:
> Suggested phrasing (how do I do this in this github UI directly?): To avoid starving other teams from judgehost resources, submissions from teams that submit while they have other submissions in the judge queue will be penalized by downgrading their judging priority.

From the files tab, close to the line number you can drag. than you can comment there and there is a symbol [page with plus-minus] where you can change your suggestion.